props-mongodb/src/testFixtures/java/sh/props/mongodb/testfixtures/MongoDbFixtures.java

```
@@ -0,0 +1,133 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 - 2021 Mihai Bojin
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+package sh.props.mongodb.testfixtures;
+
+import static java.lang.String.format;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.ReadConcern;
+import com.mongodb.ReadPreference;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import java.util.Base64;
+import java.util.Random;
+import org.bson.Document;
+
+public class MongoDbFixtures {
+
+  /**
+   * Creates a connection string using any values provided via System Properties (<code>hosts</code>
+   * ); defaults to <code>127.0.0.1:27017</code>.
+   *
+   * @return a connection string
+   */
+  public static String connectionString(int port) {
+    String connString =
+        "mongodb://127.0.0.1:%d/?maxPoolSize=2&serverSelectionTimeoutMS=2000&connectTimeoutMS=2000"
```

configure low timeouts, ensuring the tests fail fast